<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_Up</class>
 <widget class="QWidget" name="Sign_Up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font: 9pt &quot;Sitka&quot;;
    background-color: rgb(219, 232, 235);
    font-family: &quot;Sitka&quot;;
}

QLabel {
    font-size: 18px;
    font-weight: bold;
    color: #222;
}

QLineEdit {
    border: 2px solid #ccc;
    border-radius: 5px;
    padding: 8px;
    font-size: 16px;
    width: 250px;
}

QPushButton {
	background-color: rgb(37, 99, 235);
    color: white;
    border-radius: 5px;
    padding: 10px 20px;
    font-size: 18px;
}

QPushButton:hover {
    background-color: rgb(25, 80, 200); 
    color: white; 
}</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>650</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #64748B;
    padding: 8px;
    font-size: 12px;
    width: 200px;</string>
   </property>
   <property name="text">
    <string>Do you have an account ?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>640</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Arial&quot;;
    background-color: rgb(219, 232, 235);
color: #2563EB;
</string>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>230</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdc">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>410</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>320</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Projet c++/revision qt/template/image.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>100</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 41, 59);
</string>
   </property>
   <property name="text">
    <string>Your name and your last name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>370</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #1E293B;</string>
   </property>
   <property name="text">
    <string>Confirme your  password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>730</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #64748B;
    padding: 8px;
    font-size: 12px;
    width: 200px;</string>
   </property>
   <property name="text">
    <string>Â© 2025 Acme, Inc. All rights reserved.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>140</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>190</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 41, 59);
</string>
   </property>
   <property name="text">
    <string>Your email adress:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 41, 59);
</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="img2">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>40</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border-radius: 5px;
    padding: 5px 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../Downloads/pngegg (1).png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>280</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #1E293B;</string>
   </property>
   <property name="text">
    <string>Your password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b1">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>580</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgb(37, 99, 235);

</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>460</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #1E293B;</string>
   </property>
   <property name="text">
    <string>Poste:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="post">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>500</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>Agriculture</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IT</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="b1_2">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;
font: 9pt &quot;Segoe UI&quot;;
background-color: rgb(37, 99, 235);

</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="pwd_2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>280</y>
     <width>211</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #1E293B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="email_2">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>230</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #1E293B;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
